src/main/resources/config/config.yaml

```diff
@@ -59,4 +59,4 @@ jpa:
   cacheSharedDefault: false
 
 baseUrl: ${BASE_URL:-http://localhost:8080}
-frontendUrl: ${FRONTEND_URL:-http://localhost:3000/pay}
+productsUiPayUrl: ${PRODUCTSUI_PAY_URL:-http://localhost:3000/pay}
```

Comment:
- Minor. PRODUCTSUI or PRODUCTS_UI...
- Has this var been added to pay-ansible?

Reply:
This is already in the provision and deploy pipeline. I have to admit is a poor naming :(